tests/unit/hubble/test_hubio.py

```diff
@@ -237,8 +250,8 @@ def _mock_post(url, data, headers=None, stream=True):
         info.value
     )
 
-
-def test_push_with_authorization(mocker, monkeypatch, auth_token):
+@pytest.mark.parametrize('build_env', ['DOCSQA_LIB_TOKEN=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'])
```

Comment:
please change these `TOKEN`s. Explicity saying test dummy token is better. Something like `test_token...`